presto-tests/src/main/java/com/facebook/presto/tests/QueryAssertions.java

```diff
@@ -341,28 +341,75 @@ public static void copyTpchTables(
             String sourceSchema,
             Session session,
             Iterable<TpchTable<?>> tables,
-            boolean ifNotExists)
+            boolean bucketed)
     {
         log.info("Loading data from %s.%s...", sourceCatalog, sourceSchema);
         long startTime = System.nanoTime();
+
         for (TpchTable<?> table : tables) {
-            copyTable(queryRunner, sourceCatalog, sourceSchema, table.getTableName().toLowerCase(ENGLISH), session, ifNotExists);
+            copyTables(queryRunner, sourceCatalog, sourceSchema, session, table.getTableName().toLowerCase(ENGLISH), false, bucketed);
         }
         log.info("Loading from %s.%s complete in %s", sourceCatalog, sourceSchema, nanosSince(startTime).toString(SECONDS));
     }
 
-    private static void copyTable(QueryRunner queryRunner, String sourceCatalog, String sourceSchema, String sourceTable, Session session, boolean ifNotExists)
+    public static void copyTables(
```

Comment:
Rename back to `copyTable` as it's just copying one table

Reply:
fixed